templates/workshoplist.html

```diff
@@ -31,7 +31,11 @@ <h2>{{ entry.year }}</h2>
                   </td>
                   <td>
                     {% if workshop.is_publicly_visible %}
-                      <a href="{% url 'workshop_page' workshop.name %}">{{ workshop.title }}</a>
```

Comment:
Wait, I'm confused now. It was supposed to be like this:
* Your workshops - editing page
* All workshops - editing page
* Program - public page

but you made it this:
* Your workshops - editing page
* All workshops - public page
* Program - public page

This makes no sense, the admin pages were supposed to default to editing, "Program" has links to public pages, why would you want to just view the public page from an admin panel

Reply:
This is actually correct.

Editing is part of #66. Currently even superadmins don't have the permission to edit anything in other users workshops. (unless they edit through Django admin panel) See implementation of `views.can_edit_workshop` function.
Under the implementation before this PR, admins would get the public page after clicking on link in all workshops.

Reply:
Ah, okay, makes sense - I forgot there is no editing part yet.